tests/test_file.py

```diff
@@ -33,9 +36,16 @@ def test_read(self):
             with self.subTest(out_class.__name__):
                 test_read(out_class, folder=rsrcs_out, testcase=self)
 
+                # Recently added (#20)
+                test_read(out_class, folder=rsrcs_out_cassava, testcase=self)
+
 
 class TestFinalOutputs(unittest.TestCase):
     def test_read(self):
         for final_out_class in final_out_classes:
             with self.subTest(final_out_class.__name__):
                 test_read(final_out_class, folder=rsrcs_out, testcase=self)
+
+                # Recently added (#20)
+                test_read(final_out_class, folder=rsrcs_out_cassava, testcase=self)
+
```

Comment:
Hello,

I'm fairly sure that this should work without the additional `test_read`, because of the `os.walk` in the `test_read` function. As long as `Cassava` is a subfolder of `mock_DSSAT/OUT`.

Thanks!
`for root, dirs, files in os.walk(folder):`